### DISCUSSION TITLE
Кольцо Лютеро

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#364] [General]
ItRi_Prot_Total_01 - кольцо непобедимости (3/3/2/2)
ItRi_Prot_Total_02 - кольцо неуязвимости (5/5/3/3)

Лютеро предлагает кольцо неуязвимости за когти снеппера:
```cpp
AI_Output(self,other,"DIA_Lutero_Hello_13_07");  //Я могу дать тебе кольцо неуязвимости.
```
Но по факту дает кольцо непобедимости (ItRi_Prot_Total_01).

Варианты решения:

1. поменять местами названия колец
2. давать ГГ обещанное кольцо неуязвимости ItRi_Prot_Total_02
3. обрезать реплику до "Я могу дать тебе кольцо"

Все эти варианты мне одинаково не нравятся, так что это просто "на подумать".

**Comment by Kor-Angar** [comment]
проблема опять же касается готики 2 классик. там такого разнообразие колец не было и не было сетовых колец. поэтому кольцо ItRi_Prot_Total_02  было топовое и герой в нём ходил всю игру. поэтому вариант 2 плохой, так как затронет баланс. 
перебить название колец самый простой вариант. 

ввиду того что топик "про подумать" давай смотреть название колец в немецкой версии и смотреть что там говорит Лютеро на немецком. возможно что это как то связано с переводом.

ItRi_Prot_Total_01  в одном же экземпляре?  может его просто назвать "Кольцо Лютеро"?

**Comment by dosinabox** [comment]
Применимо и к классике, и к английским версиям аддона и классики. Там такая же неправильная реплика. Немецкие версии не смотрела.

> перебить название колец самый простой вариант

Да, но есть гайды и статьи в интернете, которые внезапно станут несоответствующими действительности) И всё из-за одной реплики, которая даже не для всех гильдий.

**Comment by dosinabox** [comment]
```cpp
AI_Output (self, other, "DIA_Lutero_Hello_13_07");	//I could get you a ring of invincibility.
```
ItRi_Prot_Total_01 - Ring of Inconquerability
ItRi_Prot_Total_02 - Ring of Invincibility

**Comment by Kor-Angar** [comment]
зачем на английскую версию ссылаться? :-) в ней ошибок при переводе не меньше чем в Русской ;-) 

***

обратимся к первоисточнику:

_AI_Output (self ,other,"DIA_Lutero_Hello_13_07"); //Ich kann dir einen Ring der **Unbezwingbarkeit** besorgen.
Ring der **Unbezwingbarkeit** +3 Schutz vor Waffen, +2 Schutz vor Magie und Feuer_
тут всё корректно.

Ring der Unbezwingbarkeit хз как переводится, яндекс говорить что это "кольцо неукротимости".

**Comment by dosinabox** [comment]
> зачем на английскую версию ссылаться? :-) в ней ошибок при переводе не меньше чем в Русской ;-)

Немецкой не было под рукой.

> тут всё корректно.

Значит, неправильно перевели в английской версии, а оттуда дословно перекочевало в русскую.
Видимо, придется просто с этим жить)

**Comment by Kor-Angar** [comment]
в аддоне это кольцо по моему есть так же  и в продаже. хотя точно не уверен.
